src/frontend/magnify/src/components/rooms/AddGroupToRoomDialog/AddGroupToRoomDialog.tsx

```diff
@@ -0,0 +1,147 @@
+import { defineMessages, useIntl } from 'react-intl';
+import React, { useState } from 'react';
+import { Box, Button, Heading, Layer, Select, Text } from 'grommet';
+import { FormClose } from 'grommet-icons';
+import { useMutation, useQuery } from 'react-query';
+
+import { useController } from '../../../controller';
+import { Group } from '../../../types/group';
+import { LoadingButton, WaitingRow } from '../../design-system';
+
+const messages = defineMessages({
+  AddGroupToRoomDialogTitle: {
+    id: 'components.rooms.AddGroupToRoomDialog.AddGroupToRoomDialogTitle',
+    defaultMessage: 'Add groups to room',
+    description: 'Title of the add group to room dialog',
+  },
+  selectGroupsToAdd: {
+    id: 'components.rooms.AddGroupToRoomDialog.selectGroupsToAdd',
+    defaultMessage: 'Select groups to add',
+    description: 'Label for the select groups to add',
+  },
+  addGroupsLabel: {
+    id: 'components.rooms.AddGroupToRoomDialog.addGroupsLabel',
+    defaultMessage:
+      'Add {numberSelected} {numberSelected, plural, =0 {group} one {group} other {groups}}',
+    description: 'Label for the add groups button',
+  },
+  cancelLabel: {
+    id: 'components.rooms.AddGroupToRoomDialog.cancelLabel',
+    defaultMessage: 'Cancel',
+    description: 'Label for the cancel button',
+  },
+});
+
+const renderGroup = (group: Group | undefined, onRemove: (id: string) => void) => {
+  if (!group) return null;
+  return (
+    <Button
+      key={group.id}
+      onClick={(event: React.MouseEvent) => {
+        event.preventDefault();
+        event.stopPropagation();
+        onRemove(group.id);
+      }}
+      href="#"
+      onFocus={(event: React.FocusEvent) => event.stopPropagation()}
```

Comment:
Several questions here. Why do we need:
- to stop the click and focus event propagation
- to specify this href

Reply:
This is the suggestion from grommet. 

As I understand it, we need to capture these events to stop the propagation to the select component (that would close the selector)

If we don't specify the href, it's considered a "button", and a "button" shouldn't be child of another button, but the select component of grommet is a button. Setting the "href" make it a link, and then it's valid.

You can see the example from grommet [here](https://storybook.grommet.io/?path=/story/input-select-children--children)

Would you have an alternative ?